<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>500</width>
                <height>519</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Best Painter - Score</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QPushButton" name="find_player_btn">
                <property name="geometry">
                    <rect>
                        <x>110</x>
                        <y>70</y>
                        <width>281</width>
                        <height>81</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>14</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>Найти игрока</string>
                </property>
            </widget>
            <widget class="QPushButton" name="get_table_btn">
                <property name="geometry">
                    <rect>
                        <x>110</x>
                        <y>200</y>
                        <width>281</width>
                        <height>91</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>14</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>Посмотреть таблицу игроков</string>
                </property>
            </widget>
            <widget class="QPushButton" name="get_picture_table_btn">
                <property name="geometry">
                    <rect>
                        <x>110</x>
                        <y>330</y>
                        <width>281</width>
                        <height>91</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <pointsize>14</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>Посмотреть таблицу картинок</string>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>500</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
